addOns/ascanrulesAlpha/src/test/java/org/zaproxy/zap/extension/ascanrulesAlpha/HiddenFilesScanRuleUnitTest.java

```diff
@@ -310,13 +465,28 @@ protected Response serve(IHTTPSession session) {
         }
     }
 
-    private static class OkResponseWithReqPath extends StaticContentServerHandler {
+    private static class OkResponse extends StaticContentServerHandler {
 
         private static final String OK_RESPONSE =
                 "<!DOCTYPE html\">\n" + "<html><head></head><H>Site Title</H1>Site Text... <html>";
 
-        public OkResponseWithReqPath(String path) {
+        public OkResponse(String path) {
             super(path, OK_RESPONSE);
         }
     }
+
+    private static class OkBinResponse extends StaticContentServerHandler {
+        private String content;
+
+        public OkBinResponse(String path, String content) {
+            super(path, content);
+            this.content = content;
+        }
+
+        @Override
+        protected Response serve(IHTTPSession session) {
```

Comment:
This could be removed, the content type does not matter (does no harm either but it's misleading).